test/Microsoft.DotNet.Cli.Utils.Tests/StreamForwarderTests.cs

```diff
@@ -39,16 +38,18 @@ public static IEnumerable<object[]> ForwardingTheoryVariations
                 return new[]
                 {
                     new object[] { "123", new string[]{"123"} },
-                    new object[] { "123\n", new string[] {"123"} },
-                    new object[] { "123\r\n", new string[] {"123"} },
+                    new object[] { "123\n", new string[] {"123", ""} },
```

Comment:
Hmm this reasoning makes sense, but I'm wondering about this:

`s_expectedOutput = "Hello World!" + Environment.NewLine + Environment.NewLine;`

The app that one is executing just uses Console.WriteLine, so we should only expect to see one newline don't you think?